### PR TITLE
[BO] - 11102 - edit drag and drop page position. New design 

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | edit drag and drop page position. New design on position page drag and drop element.
| Type?         | improvement
| Category?     |  BO 
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | (optional) If this PR fixes an [Issue](https://github.com/PrestaShop/PrestaShop/issues), please write "Fixes #[issue number]" here.
| How to test?  | Please indicate how to best verify that this PR is correct.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12628)

### CODE REVIEW

File: admin-dev/themes/default/sass/partials/_header.sass [hunk 1]
```diff
@@ -211,7 +211,7 @@ $search-border-color: #bbcdd2
 
 			#employee_links
 				.employee_avatar
-					display: none !important
+					// display: none !important
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/sass/partials/_header.sass:214]: remove the line instead please

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 2]
```diff
@@ -499,21 +499,85 @@ $header-text-color: #4e6167;
     }
   }
   .dropdown-menu {
-    top: 0.5rem !important;
+    top: 0.9rem !important;
     left: 0.3rem !important;
     margin: 0;
-    min-width: 10.625rem;
+    // min-width: 10.625rem;
```
Comment (mickaelandrieu) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss:505]: remove the line instead

**Review by mickaelandrieu — CHANGES_REQUESTED** [review]
css files needs to be updated and assets built again

**Comment by colinegin** [comment]
#11102

**Comment by Quetzacoalt91** [comment]
Hi there, did you mean `develop` as the target branch?

File: admin-dev/themes/default/public/bundle.js [hunk 3]
```diff
@@ -1,2 +1,134 @@
-/******/!function(e){// webpackBootstrap
-/******/var t={};function n(r){if(t[r])return t[r].exports;var o=t[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,n),o.l=!0,o.exports}n.m=e,n.c=t,n.d=function(e,t,r){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:r})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(n.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)n.d(r,o,function(t){return e[t]}.bind(null,o));return r},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.p="",n(n.s=0)}([function(e,t,n){e.exports=n(1)},function(e,t,n){"use strict";n.r(t);n(2),n(4)},function(e,t,n){},,function(e,t,n){}]);
\ No newline at end of file
+/******/ (function(modules) { // webpackBootstrap
```
Comment (PierreRambaud) [inline admin-dev/themes/default/public/bundle.js:1]: This is the dev file, you must commit the prod files :)

File: admin-dev/themes/default/sass/partials/_header.sass [hunk 4]
```diff
@@ -224,27 +224,64 @@ $search-border-color: #bbcdd2
...
+					    display: inline-block
+					    margin: 0 0 12.5px
 
 			.employee_name
 				text-align: center
 
 			.dropdown-menu
 				min-width: inherit
-				width: 10.625rem
+				// width: 10.625rem
```
Comment (PierreRambaud) [inline admin-dev/themes/default/sass/partials/_header.sass:243]: Same here remove useless comment

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 5]
```diff
@@ -499,21 +499,85 @@ $header-text-color: #4e6167;
...
+        .material-icons{
+          color: $dropdown-link-hover-bg;
+        }
+      }
+      }
+    
     .employee_avatar {
       text-align: center;
-      border-bottom: 1px solid $color-separator;
+      // border-bottom: 1px solid $color-separator;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss:542]: Here too

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 6]
```diff
@@ -499,21 +499,85 @@ $header-text-color: #4e6167;
...
       padding: 0.687rem 0 1rem;
       margin-bottom: 0.46875rem;
       font-weight: 600;
       img {
-        display: none;
+        // display: none;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss:547]: same

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
You commit the dev files instead build :)

File: admin-dev/themes/default/package.json [hunk 7]
```diff
@@ -18,7 +18,7 @@
         "file-loader": "^1.1.11",
         "material-design-icons": "^2",
         "mini-css-extract-plugin": "^0.4.2",
-        "node-sass": "^4.9.3",
+        "node-sass": "^4.11.0",
```
Comment (eternoendless) [inline admin-dev/themes/default/package.json:21]: Why did you update this dependency?

File: admin-dev/themes/default/public/bundle.js (hunk 3 above)
Comment (eternoendless) [reply admin-dev/themes/default/public/bundle.js:1]: Please read the [rules on compiled assets](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#compiled-assets)

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 8]
```diff
@@ -499,21 +499,85 @@ $header-text-color: #4e6167;
...
+    top: 0.9rem !important;
     left: 0.3rem !important;
     margin: 0;
-    min-width: 10.625rem;
+    // min-width: 10.625rem;
+    min-width: 17.5rem;
     hr {
       border-top-width: 2px;
     }
+      .divider{
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss:510]: Please mind the code indentation and spacing around braces.

File: admin-dev/themes/new-theme/scss/components/_showcase-card.scss [hunk 9]
```diff
@@ -50,6 +50,12 @@
       text-align: left;
     }
 
+       > a{
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/scss/components/_showcase-card.scss:53]: Please mind the indentation and spacing around braces.

File: admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl [hunk 10]
```diff
@@ -28,16 +28,25 @@
...
+      <a class="dropdown-item employee-link profile-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}">
+      <i class="material-icons">settings</i>
       {l s='Your profile'}
     </a>
-    <a class="dropdown-item employee-link" id="header_logout" href="{$logout_link|escape:'html':'UTF-8'}">
-      <i class="material-icons">power_settings_new</i>
+    </div>
+    
+    <p class="divider"></p>
+    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17'}"><i class="material-icons">book</i> {l s='Resources'}</a>
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl:42]:
```
{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17'}
```

I'm not sure that links are supposed to be treated as wordings. What do you think @LouiseBonnard ?

Comment (eternoendless) [inline admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl:42]: New wordings like

```
{l s='Resources'}
```

must be linked to a [translation domain](https://devdocs.prestashop.com/1.7/development/internationalization/translation/translation-domains/). TBD by @LouiseBonnard

**Review by eternoendless — CHANGES_REQUESTED** [review]
What issue does this PR fix? Please complete the Pull Request form as explained in the [Contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#the-pull-request-form).

File: admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl (hunk 10 above)
Comment (LouiseBonnard) [reply admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl:42]: I agree with @eternoendless, as said earlier to @YeLnatSs. :-)

Comment (LouiseBonnard) [reply admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl:42]: Let's localize it in _Admin.Global_ (or _Admin.Navigation.Header_ if the string is already localized in this file for the new bo menu needs).

**Comment by jolelievre** [comment]
I don't understand the PR was simply closed. I won't be merged?

**Comment by PierreRambaud** [comment]
🤔

**Comment by colinegin** [comment]
It's because there is a new PR for the same issue : https://github.com/PrestaShop/PrestaShop/pull/12646

**Pull request closed without merging**